lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -56,12 +56,12 @@
       </a>
         % else:
         <a class="fade-cover">
-        <img src="${course_image_url(course)}" alt="${_('{course_number} {course_name} Cover Image').format(course_number=course.number, course_name=course.display_name_with_default) |h}" />
+        <img src="${course_image_url(course)}" alt="${_('{course_number} {course_name} Home Page').format(course_number=course.number, course_name=course.display_name_with_default) |h}" />
```

Comment:
@ahsan-ul-haq is correct here.  We cannot have a null alt on a linked image.  However, there is another solution.  We currently have 3 ways to get to a course home page for each course block (too many IMHO).  We could hide this one by adding aria-hidden=true to the anchor element so that it is removed from the accessibility tree.  The end result would be two ways to get to the course page, the Linked title/heading and the view course button.  Thoughts @clrux ?